### ISSUE TITLE
solargraph.commandPath not picking the gem from mentioned RVM location

### ISSUE THREAD

**Issue by aruprakshit** [castwide/vscode-solargraph#60]
Hi,

I do have below settings:

```json
{
  // Basic settings: turn linter(s) on
  "ruby.lint": {
    "reek": true,
    "rubocop": {
      "lint": true,
      "rails": true
    },
    "ruby": true, //Runs ruby -wc
    "fasterer": true,
    "debride": true,
    "ruby-lint": true
  },
  // Time (ms) to wait after keypress before running enabled linters. Ensures
  // linters are only run when typing has finished and not for every keypress
  "ruby.lintDebounceTime": 500,
  "ruby.format": "rubocop",
  "solargraph.formatting": true,
  "solargraph.autoformat": true,
  "solargraph.completion": true,
  "solargraph.commandPath": "/Users/aruprakshit/.rvm/gems/ruby-2.3.3@global/bin/solargraph"
}
```

But when I open the folder in my editor (VSCode), I see below error:

<img width="1216" alt="screen shot 2018-06-04 at 3 38 57 pm" src="https://user-images.githubusercontent.com/3428463/40911965-e5a2672e-680d-11e8-8237-3ed4fdbfb402.png">

**Comment by aruprakshit** [comment]
@castwide I already hardcoded it as you see `  "solargraph.commandPath": "/Users/aruprakshit/.rvm/gems/ruby-2.3.3@global/bin/solargraph"`, but still it is not picking up.

**Comment by castwide** [comment]
That's why I suggested a .ruby-version file. The error indicates that VS Code is correctly running the solargraph script in 2.3, but it's using the 2.4 interpreter. Testing from the command line on CentOS, I got the same error using an absolute path to an installation of solargraph that didn't match the version of Ruby in use via rvm. The gem's executable doesn't change the interpreter.

I can try some more tests on MacOS later. I don't think I've ever set `solargraph.commandPath` in conjunction with systems using rvm. I've only ever needed it on certain Windows setups.
